.github/workflows/mdx-lint.yml

```diff
@@ -16,6 +16,9 @@ jobs:
         with:
           node-version: '12'
 
+      - name: Lint for markdown correctness
+        run: npm i && npm run lint:markdownlint
```

Comment:
How about using yarn here too? A top-level `yarn.lock` file will save us from accidental breaking changes in newly released dependencies.

Reply:
We would need to install `yarn` beforehand for that. I think we're using `npm` elsewhere in our CI too, but I might be wrong.

CC: @CiaranMn

Reply:
I guess we can add a follow-up task to bring all dependencies to the parent `package.json` and utilize this package: https://github.com/marketplace/actions/github-action-for-yarn